casper/src/main/scala/coop/rchain/casper/genesis/contracts/ProofOfStake.scala

```diff
@@ -14,7 +15,16 @@ final case class ProofOfStake(
     quarantineLength: Int = 50000
 ) extends CompiledRholangTemplate(
       "PoS.rhox",
-      NormalizerEnv.Empty,
+      NormalizerEnv(
```

Comment:
There is a pre-existing ticket to remove generated data (`RegistrySigGen` stuff) from each of the Rholang libraries, but is out of scope for this PR. That includes `PoS` spec. That would effectively introduce `PublicKey` as a parameter here instead of a constant.

Reply:
Okay. Then we should add a todo comment above the `PublicKey` that links to the ticket.